liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/AbstractLiquibaseMojo.java

```diff
@@ -654,7 +654,7 @@ public void setPassword(String password) throws Exception {
      *
      * @parameter property="liquibase.dbclHistoryEnabled"
```

Comment:
Would changing this javadoc parameter do the same thing?

Reply:
Interestingly, no. It matches on the name the Java field, or the key, not the property name. This is code that Liquibase wrote, which honestly could probably be improved in other ways.